plugins/modules/openssl_pkcs12.py

```diff
@@ -342,8 +359,9 @@ def load_certificate_set(filename, backend):
     '''
     Load list of concatenated PEM files, and return a list of parsed certificates.
     '''
-    with open(filename, 'rb') as f:
-        data = f.read().decode('utf-8')
+    if filename:
+        with open(filename, 'rb') as f:
+            data = f.read().decode('utf-8')
     return [load_certificate(None, content=cert.encode('utf-8'), backend=backend) for cert in split_pem_list(data)]
```

Comment:
Where does `data` come from if `filename` is falsy?

(I'm surprised the linters aren't reporting anything.)